docs-client/src/containers/MethodPage/DebugPage.tsx

```diff
@@ -116,6 +117,17 @@ const copyTextToClipboard = (text: string) => {
   modal.removeChild(textArea);
 };
 
+const parseServerRootPath = (docServiceRoute: string) => {
+  const plainRoute = docServiceRoute.replace('/*', '');
+  const index = window.location.href.indexOf(plainRoute);
+
+  if (index === -1) {
+    return '';
+  }
+  const rootPath = window.location.href.substring(0, index);
+  return rootPath;
```

Comment:
Question) I haven't tried, but what do you think of returning a relative path instead of the full location? I feel like this can be a confusing point later if either an absolute path or a relative path can be set to `additionalPath`

i.e. `/foo` instead of `http://host/foo`

Reply:
Hmm good idea, I was also thinking about the same thing let me try.

Reply:
Now using a relative path. Tested manually behind a proxy and a non-proxy.